openapi/definitions.yaml

```diff
@@ -322,14 +322,42 @@ BlockedInboxOrChannels:
   description: |-
     All the notification channels blocked by the user.
     Each channel is related to a specific service (sender).
+NewProfile:
```

Comment:
why is this needed? if you add `is_email_validated` to `Profile` you should get the same result

Reply:
@cloudify 
Let me try to explain the logic of the types:

**NewProfile**: contains is_email_validated, this field is needed because the Backend on first user login CREATE the profile and set is_email_validated to true if the login is done with SPID and to false otherwise.

**Profile**: do NOT contains is_email_validated because i don't want the Backend nor the App to directly manipulate this field, only the verification process can change it. An alternative is to accept it in the type and ignore it on update but i think it is a violation of the "contract" established through the specification.

**ExtendedProfile**: contains is_email_validated because we want to return ALL the profile information to the Backend and than the App.

Reply:
ok makes sense